service/README.md

```diff
@@ -53,17 +53,87 @@ Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/configurer/utils/fp
+|auth.provider.service.client.baseUrl|ENV|Configuration required by auth-checker-library to communicate with s2s auth service|
+|spring.mail.host|ENV|Mail server host used for robotics (non Gov notify emails|
+|spring.mail.port|ENV|Port of the mail server|
+|spring.mail.properties.mail.smtp.starttls.enable|ENV|Is mail TLS enabled|
+|spring.mail.properties.mail.smtp.ssl.trust|ENV|TLS mail config|
+|spring.security.enabled|ENV|Indicates if spring security should be enabled|
+|spring.security.oauth2.resourceserver.jwt.issuer-uri|ENV|Expected IDAM issuer-uri|
+|spring.security.oauth2.resourceserver.jwt.jwk-set-uri|ENV|Expected IDAM jwk-set-uri|
+|core_case_data.api.url|ENV|CCD Data store API URL|
+|docmosis.tornado.url|ENV|Docmosis Tornado (document generation service) URL|
+|docmosis.tornado.key|SECRET|Docmosis Tornado (document generation service) API key|
+|document_management.url|ENV|URL for service responsible for storing documents|
+|notify.api_key|SECRET|Gov notify API Key for sending emails|
+|gateway.url|ENV|URL for CCD gateway, currently used to prepare URL leading directly to the document, used in notifications|
+|rd_professional.api.url|ENV|URL for Professional Reference Data service|
+|send-letter.url|ENV|URL for service delivering Bulk Print functionality|
+|ccd.ui.base.url|ENV|URL for CCD Web UI, used in notifications|
+|fpl.local_authority_codes_forbidden_case_submission|ENV|(Optional) Local authority code, which will be blocked from submitting applications|
+|fpl.local_authority_email_to_code.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_name.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_hmcts_court.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_cafcass.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_shared_inbox.mapping|SECRET|Explained below.|
+|fpl.local_authority_fallback_inbox|SECRET|Fallback notification inbox when system cannot determine where LA notification should be delivered|
+|fpl.local_authority_fallback_inbox|SECRET|Fallback notification inbox when system cannot determine where LA notification should be delivered|
+|fpl.system_update.username|SECRET|System user username, used for automated state transitions and data modifications|
+|fpl.system_update.password|SECRET|System user password, used for automated state transitions and data modifications|
+|fpl.ctsc_inbox|SECRET|CTSC mail inbox|
+|robotics.notification.sender|SECRET|FROM field when sending emails to robotics|
+|robotics.notification.recipient|SECRET|Robotics mailbox address|
+|feature.toggle.robotics.case-number.notification.enabled|ENV|Determines if JSON file should be send to robotics when Family Man case number is added to the case'
+|feature.toggle.robotics.support.api.enabled|ENV|Enables API to retrigger robotics notification for particular case|
+|appinsights.instrumentationkey|SECRET|Key used to connect to Azure AppInsights|
+
+Two notes:
```

Comment:
I would just say Notes, don't bother with the 2, if we then need to add more it is just another thing that might be missed.

Reply:
Done! Good point.